src/gtk/spinctrl.cpp

```diff
@@ -145,6 +145,10 @@ bool wxSpinCtrlGTKBase::Create(wxWindow *parent, wxWindowID id,
 
     PostCreation(size);
 
+    // Fix the min width because PostCreation sets it to `size`, but the
+    // control can be thinner.
+    SetMinSize(GetSizeFromText("9"));
+
```

Comment:
No, this is wrong. The initial size must become the min size. If you don't want it to, don't create the control with this size.

But initial size is the min size by convention, if specified.